src/localMetaConfiguration.h

```diff
@@ -21,26 +21,29 @@
 
 #include "metaIOConfig.h"
 
-#if defined(METAIO_FOR_ITK) || !defined(METAIO_FOR_VTK)
+#if defined(METAIO_FOR_ITK)
 // ITK
 
 #  define METAIO_USE_NAMESPACE 0
 #  define METAIO_NAMESPACE ITKMetaIO
+#  define METAIO_STREAM std
 
-#  include "itk_zlib.h"
+#  include <itk_zlib.h>
 
 #  include <iostream>
 #  include <fstream>
 
 #  define METAIO_EXPORT
 
-#else
+#elif defined(METAIO_FOR_VTK)
 // VTK
 
 #  define METAIO_USE_NAMESPACE 1
```

Comment:
Why is the namespace 1 defined for VTK?

Reply:
This is historic/legacy and unchanged.   VTK no longer includes MetaIO.  We could remove all reference to VTK, but perhaps VTK will change their mind and include MetaIO once again and this will be a good starting point.   Otherwise, removing all reference to VTK should probably happen in a separate pull request.